liquidprompt

```diff
@@ -3835,6 +3849,113 @@ _lp_temperature_color() {
     lp_temperature_color="${LP_MARK_TEMP}${ret}${lp_temperature}°${NO_COL}"
 }
 
+######################
+# Current Disk Usage #
+######################
+
+# Convert bytes in human-readable, binary-prefixed notation (using powers of 2).
+# Precision can be 0, 1 or 2 digits. If not given, precision is two digits.
+__lp_bytes_to_human() { # bytes, [precision] -> lp_bytes, lp_bytes_units
+    local i="$1" prec="${2:-2}" size remainder=""
+    # This has to be split on multiple lines for Zsh 5.0 (see wiki)
+    local units=()
```

Comment:
Despite the note, this is not the correct way to keep this from breaking on Zsh 5.0. See [the wiki](https://github.com/nojhan/liquidprompt/wiki/Shell-standards#quoting) again.
> __lp_bytes_to_human:7: units[@]: parameter not set

```suggestion
    local -a units
```